testgrid/config.yaml

```diff
@@ -3816,330 +3816,330 @@ dashboards:
-  - name: gke-gci-1.9-gci-1.10-upgrade-cluster
+  - name: gke-gci-1.10-gci-1.11-upgrade-cluster
     test_group_name: ci-kubernetes-e2e-gke-gci-stable2-gci-stable1-upgrade-cluster
-  - name: gke-gci-1.9-gci-1.10-upgrade-cluster-new
+  - name: gke-gci-1.10-gci-1.11-upgrade-cluster-new
     test_group_name: ci-kubernetes-e2e-gke-gci-stable2-gci-stable1-upgrade-cluster-new
-  - name: gce-1.9-1.10-gci-kubectl-skew
+  - name: gce-1.10-1.11-gci-kubectl-skew
     test_group_name: ci-kubernetes-e2e-gce-stable2-stable1-gci-kubectl-skew
-  - name: gce-1.10-1.9-gci-kubectl-skew
+  - name: gce-1.11-1.10-gci-kubectl-skew
     test_group_name: ci-kubernetes-e2e-gce-stable1-stable2-gci-kubectl-skew
-  - name: gke-1.10-1.9-gci-kubectl-skew
+  - name: gke-1.11-1.10-gci-kubectl-skew
     test_group_name: ci-kubernetes-e2e-gke-stable1-stable2-gci-kubectl-skew
-  - name: gke-1.9-1.10-gci-kubectl-skew
+  - name: gke-1.10-1.11-gci-kubectl-skew
     test_group_name: ci-kubernetes-e2e-gke-stable2-stable1-gci-kubectl-skew
-  - name: gce-1.9-1.10-gci-kubectl-skew-serial
+  - name: gce-1.10-1.11-gci-kubectl-skew-serial
     test_group_name: ci-kubernetes-e2e-gce-stable2-stable1-gci-kubectl-skew-serial
-  - name: gce-1.10-1.9-gci-kubectl-skew-serial
+  - name: gce-1.11-1.10-gci-kubectl-skew-serial
     test_group_name: ci-kubernetes-e2e-gce-stable1-stable2-gci-kubectl-skew-serial
-  - name: gke-1.10-1.9-gci-kubectl-skew-serial
+  - name: gke-1.11-1.10-gci-kubectl-skew-serial
     test_group_name: ci-kubernetes-e2e-gke-stable1-stable2-gci-kubectl-skew-serial
-  - name: gke-1.9-1.10-gci-kubectl-skew-serial
+  - name: gke-1.10-1.11-gci-kubectl-skew-serial
     test_group_name: ci-kubernetes-e2e-gke-stable2-stable1-gci-kubectl-skew-serial
-  - name: gce-1.9-1.10-upgrade-master
+  - name: gce-1.10-1.11-upgrade-master
     test_group_name: ci-kubernetes-e2e-gce-stable2-stable1-upgrade-master
-  - name: gce-1.9-1.10-upgrade-cluster
+  - name: gce-1.10-1.11-upgrade-cluster
     test_group_name: ci-kubernetes-e2e-gce-stable2-stable1-upgrade-cluster
-  - name: gce-1.9-1.10-upgrade-cluster-new
+  - name: gce-1.10-1.11-upgrade-cluster-new
     test_group_name: ci-kubernetes-e2e-gce-stable2-stable1-upgrade-cluster-new
-  - name: kubelet-1.10
+  - name: node-kubelet-1.11
     test_group_name: ci-kubernetes-node-kubelet-stable1
-  - name: gce-gpu-1.10
+  - name: gce-gpu-1.11
     test_group_name: ci-kubernetes-e2e-gce-gpu-stable1
-  - name: gce-device-plugin-gpu-1.10
+  - name: gce-device-plugin-gpu-1.11
     test_group_name: ci-kubernetes-e2e-gce-device-plugin-gpu-stable1
-  - name: gke-device-plugin-gpu-1.10
+  - name: gke-device-plugin-gpu-1.11
     test_group_name: ci-kubernetes-e2e-gke-device-plugin-gpu-stable1
-  - name: gke-device-plugin-gpu-p100-1.10
+  - name: gke-device-plugin-gpu-p100-1.11
     test_group_name: ci-kubernetes-e2e-gke-device-plugin-gpu-p100-stable1
 
 - name: sig-release-1.10-blocking
   dashboard_tab:
-  - name: build-1.10
+  - name: build-1.11
```

Comment:
If we're going to bump the dashboards like this we'll also need to bump the dashboard name.